homework/style.css

```diff
@@ -1,3 +1,140 @@
+/* will improve css further in the next commit */
```

Comment:
On a mobile, the fonts get tiny: better not scale them.

![image](https://user-images.githubusercontent.com/2788771/62366409-0ee8ca80-b527-11e9-9253-c7783db39f20.png)

Reply:
fixed and got rid of the logo since it created more problems while scaling

Reply:
Your layout still leaves room for improvement.

Mobile:

![image](https://user-images.githubusercontent.com/2788771/62689300-d3d51400-b9ca-11e9-8edf-8de1617b0b3a.png)

On a wide-screen it would be better to limit the content width rather than stretching it out to the full width:

![image](https://user-images.githubusercontent.com/2788771/62689387-0121c200-b9cb-11e9-90d5-2bfd9dadbe1b.png)